### DISCUSSION TITLE
TerriaMap interface translation

### DISCUSSION THREAD

**Discussion by LorenzoStucchi** [TerriaJS/terriajs#5522] [Ideas] [answered]
Dear developers,

I saw that it is possible to contribute to the translation of the terriajs on weblate, is this just the translation of it or it will affect also the interface of TerriaMap?

I briefly check the code and I saw that TerriaMap is available only in English it is correct? 

Thanks 😃

**Comment by arky** [comment, marked as answer]
Yes, You can contribute translations here https://hosted.weblate.org/projects/terriajs/ . We are still waiting for the developers to pull and ship the languages in the main source repositories. 

Here is the example of terrriajs  localized in Khmer (km). https://data.opendevelopmentcambodia.net/km/map-explorer

**Reply by AnaBelgun** [reply]
Correct, thank you @arky

**Reply by LorenzoStucchi** [reply]
Thanks, we started the translation into Italian. We are planning to provide the full translation for August-September

**Comment by irbian** [comment]
hey, any news on this?
